### PR TITLE
71 conecting tasktree UI to backend

### PR DESCRIPTION
**Pull request by Bhan-Aayush**
I finally finished connecting my UI to the backend, it was very difficult however thanks to David and Marcial's assistance the code was finally able to work. I have left the TODOs for connecting other parts of the code, which I am going to do tomorrow by inserting Oswin's UI.

### CODE REVIEW

File: src/main/java/task_tree_UI/TaskTreeUI.java [hunk 1]
```diff
@@ -21,25 +22,37 @@
...
+    private final DisplayTaskTreeController displayController;
+    private final WindowManager programWindows;
 
 
     /**
      * Default constructor for TaskTree UI
      */
-    public TaskTreeUI() {
+    public TaskTreeUI(WindowManager windowManager, DisplayTaskTreeController displayController) {
+        this.programWindows = windowManager;
```
Comment (dadler83) [inline src/main/java/task_tree_UI/TaskTreeUI.java:40]: You also have to add this window to the windowManager, and create a reference key for it, so other windows can access it

**Review by dadler83 — COMMENTED** [review]
Looks good, glad this is done, but you need to add docstrings, and finish connecting this to the window manager, so other windows can use this window.

**Review by bmguiler — APPROVED** [review]

**Review by dadler83 — APPROVED** [review]
Looks good, thanks for making the changes

**Review by MMachadoUofT — APPROVED** [review]
Excellent work here, Aayush! I'm glad we were able to finish up the documentation in this PR. Now the whole tasktree aspect of this program should be fully functional!

**Pull request merged**